### DISCUSSION TITLE
Doubts about free energy calculation of ice

### DISCUSSION THREAD

**Discussion by Ding-Hairui** [deepmodeling/deepmd-kit#1859] [Q&A] [answered]
Dear maintainers,
  I met some problems when repeating your works on water phase diagram (PhysRevLett.126.236001).
  As mentioned in supplymentary materials, the phase boundary point of ice-Ih and ice-II is about [150 K, 0.35 GPa]. I follow the steps of your DPTI project (https://github.com/deepmodeling/dpti), and perform three-step hamiltonian thermodynamic integration calculations at [150K, 0.35GPa] and thermodynamic integration calculations from 0.33GPa to 0.37GPa for thses two phase. Then I found the calculated free energies of them are too much different.
![Figure_1](https://user-images.githubusercontent.com/47772564/184563838-84d361cc-b309-4f8d-9487-cf991dd3608b.png)
  The ice structures and DP model are from dplibrary (https://dplibrary.deepmd.net/#/project_details?project_id=202206.001). And most parameters are the same as the dpti/test/benchmark_hti_ice and benchmark_ti_water. So there should not be such error. I can not find where the problems are. Attached are my input files. The output files are too large to upload. Could you please tell me how to solve it?
[inputs.zip](https://github.com/deepmodeling/deepmd-kit/files/9335622/inputs.zip)

**Comment by wanghan-iapcm** [comment, marked as answer]
One may follow the section 9 of this work http://iopscience.iop.org/0953-8984/20/15/153101 to check the calculation of free energies.

**Reply by Ding-Hairui** [reply]
Sorry for my late reply. I finally found where the mistake is: I input a wrong number of atoms. Now the result agrees with your work!

**Reply by AnuragKr** [reply]
Hello @nyanss,

Hope you are doing well. Could you please tell me how you got the ice Ih initial structure for the LAMMPS simulation?  

Thanks

**Reply by Ding-Hairui** [reply]
@AnuragKr The initial structures I used are from dplibrary (http://dplibrary.deepmd.net/#/project_details?project_id=202010.001) which is mentioned at the reference of their article (Phys. Rev. Lett. 126, 236001 (2021)).
